python/ray/worker.py

```diff
@@ -748,7 +748,8 @@ def init(
     for hook in _post_init_hooks:
         hook()
 
-    return _global_node.address_info
+    node_id = global_worker.core_worker.get_current_node_id()
```

Comment:
Why do we need to include this in `address`?

Reply:
Mainly for testing, and the `node id` in address is to show the owner of the address.